manifests/d/Discord/Discord/Development/1.0.670/Discord.Discord.Development.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Discord.Discord.Development
+PackageVersion: 1.0.670
+PackageLocale: en-US
+Publisher: Discord Inc.
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Discord Development
+PackageUrl: https://discord.com/
+License: (c) 2021 Discord Inc. All rights reserved.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate